### ISSUE TITLE
Can reach the x button to clear TextBlock content by only keyboard

### ISSUE THREAD

**Issue by acouvert-msft** [microsoft/microsoft-ui-xaml#7949]
### Describe the bug

While editing the content of a `TextBlock`, x button appears on the right. Clicking on this button clears the `TextBlock `content. Unfortunately, this button is not reachable without a mouse. There is alternative to clear the content by only using the keyboard, but I was wondering if having this button not reachable could be considered as an accessibility issue.

### Steps to reproduce the bug

1. Create a sample app with a `TextBlock`
2. Start to edit `TextBlock `content
3. A x button should appear on the right
4. Try to reach this button by only using the keyboard 

### Expected behavior

I'm expecting to be able to reach, by only using a keyboard, that x button to clear a `TextBlock `content.

### NuGet package version

WinUI 2 - Microsoft.UI.Xaml 2.8.1

**Comment by kmahone** [comment]
This is an intentional design choice for TextBox.

The TextBox can still be cleared using only keyboard shortcuts without having to invoke the 'x' button. You can do: Ctrl+A to select all the text and then do Backspace to clear it.
